pkg/apis/query/v0alpha1/results.go

```diff
@@ -0,0 +1,64 @@
+package v0alpha1
+
+import (
+	"encoding/json"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	openapi "k8s.io/kube-openapi/pkg/common"
+	spec "k8s.io/kube-openapi/pkg/validation/spec"
+)
+
+// Wraps backend.QueryDataResponse, however it includes TypeMeta and implements runtime.Object
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type QueryDataResponse struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Backend wrapper (external dependency)
+	backend.QueryDataResponse
```

Comment:
^^^ @toddtreece -- I don't think we can avoid this, but also not sure the best approach to solve things.

I think this is a reasonable first step, but we need to find a better story for how the plugins SDK formats show up in k8s.  

Yes the openapi is a hack, but it makes everything happy.  

We can implement DeepCopy() directly in the SDK without dependency explosion, but I am not sure about TypeMeta and runtime.Object 😢